mico-core/src/main/java/io/github/ust/mico/core/web/ServiceController.java

```diff
@@ -236,28 +243,28 @@
         String localShortName = newServiceDependee.getDependedService().getShortName();
         String localVersion = newServiceDependee.getDependedService().getVersion();
         boolean dependencyAlreadyExists = (service.getDependencies() != null) && service.getDependencies().stream().anyMatch(
-                dependency -> dependency.getDependedService().getShortName().equals(localShortName)
-                        && dependency.getDependedService().getVersion().equals(localVersion));
+            dependency -> dependency.getDependedService().getShortName().equals(localShortName)
+                && dependency.getDependedService().getVersion().equals(localVersion));
         if (dependencyAlreadyExists) {
             return ResponseEntity
-                    .created(linkTo(methodOn(ServiceController.class).getServiceById(service.getId())).toUri())
-                    .body(new Resource<>(service, getServiceLinks(service)));
+                .created(linkTo(methodOn(ServiceController.class).getServiceById(service.getId())).toUri())
+                .body(new Resource<>(service, getServiceLinks(service)));
         }
 
         final MicoServiceDependency processedServiceDependee = new MicoServiceDependency()
-                .setDependedService(serviceDependeeOpt.get())
-                .setService(service);
+            .setDependedService(serviceDependeeOpt.get())
+            .setService(service);
 
         log.info("New dependency for MicoService '{}' '{}' -[:DEPENDS_ON]-> '{}' '{}'", shortName, version,
-                processedServiceDependee.getDependedService().getShortName(),
-                processedServiceDependee.getDependedService().getVersion());
+            processedServiceDependee.getDependedService().getShortName(),
+            processedServiceDependee.getDependedService().getVersion());
 
         service.getDependencies().add(processedServiceDependee);
         MicoService savedService = serviceRepository.save(service);
 
         return ResponseEntity
-                .created(linkTo(methodOn(ServiceController.class).getServiceById(savedService.getId())).toUri())
-                .body(new Resource<>(savedService, getServiceLinks(savedService)));
+            .created(linkTo(methodOn(ServiceController.class).getServiceById(savedService.getId())).toUri())
+            .body(new Resource<>(savedService, getServiceLinks(savedService)));
     }
 
     @DeleteMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}/{" + PATH_VARIABLE_VERSION + "}" + "/dependees")
```

Comment:
Consider extracting path part `dependees` to a constant.

Reply:
Added a commit to #514 (PR for issue "Refactor Service Controller")